### ISSUE TITLE
Add option to place "compose" button on bottom left

### ISSUE THREAD

**Issue by ringtailsoftware** [metabolist/metatext#161]
I hold my phone in my right hand and use my right thumb to scroll timelines. I often press the "compose toot"/post button by accident.

Provide settings option to move the button to the left of the screen.

**Comment by peterjc** [comment]
Also seems like a good idea for people who hold the phone in their left hand and want to be able to press the button with their thumb

**Comment by ryanmichaelsmith** [comment]
Yes please! I came here to create this same issue. Just make it configurable?

**Comment by denishaskin** [comment]
I never know if this is the right procedure, but I'd like to register another vote for this.  I'm accidentally ending up in the compose screen *all* the time when all I'm intending to do is scroll.

**Comment by peterjc** [comment]
Issues can get too busy with endorsement comments, so there's the option of the "thumbs up" reaction, and you can also subscribe (button at the bottom of the right hand column) to be informed about any future comments.
